nav2_default_recoveries/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+cmake_minimum_required(VERSION 3.5)
+project(nav2_default_recoveries)
```

Comment:
Lets name the package `nav2_recovery_plugins` and the original package `nav2_recovery` to be analog to the planner / controller packages now.